### PR TITLE
[not for merge] Converted reference docs to paradox

### PR DESCRIPTION
**Pull request by raboof**
Some known issues, but I think we can simply run the conversion again if
needed.

### CODE REVIEW

**Comment by patriknw** [comment]
Shall we create a `akka-docs-new` branch instead, that we work on until it's ready to be be merged into master` Then we can place/replace things at the right place (`akka-docs`) immediately and we don't mess up git history by copying the sample code around.

**Review by patriknw — CHANGES_REQUESTED** [review]
Skimmed a small fraction of it and I think it's looking good, but as mentioned in other comment I think we should not copy sample code to avoid loosing git history. Would it even be worth doing a `git mv` of the old rst files to the new location before replacing the content?

**Comment by raboof** [comment]
`git mv` for the example code seems like a good idea - and indeed then we need to use an `akka-docs-new` branch because that'd break the 'old' docs. Perhaps just merge https://github.com/pvlugter/akka/tree/wip/paradox into that, too?

Should we create issues to keep track of the various conversion TODO's?

**Comment by raboof** [comment]
As discussed on slack: good idea, I'll create a new `wip-akka-docs-new` branch and properly `git mv` stuff in there

File: akka-docs-new/src/main/paradox/additional/osgi.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+the core specification leaves these facilities up to the container in use. In turn, some containers have tooling to generate
+application load descriptors from repository metadata.
+
+## Notable Behavior Changes
+
+Combined with understanding the bundle lifecycle, the OSGi developer must pay attention to sometimes unexpected behaviors
+that are introduced. These are generally within the JVM specification, but are unexpected and can lead to frustration.
+
+ * 
+   Bundles should not export overlapping package spaces. It is not uncommon for legacy JVM frameworks to expect plugins
```
Comment (raboof) [inline akka-docs-new/src/main/paradox/additional/osgi.md]: indeed odd, though renders fine

File: akka-docs-new/src/main/paradox/common/circuitbreaker.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+   
+
+![circuit-breaker-states.png](../images/circuit-breaker-states.png)
+
+## Examples
+
+### Initialization
+
+Here's how a :class:
+*CircuitBreaker*
```
Comment (raboof) [inline akka-docs-new/src/main/paradox/common/circuitbreaker.md]: seems wrong, created #22897

File: akka-docs-new/src/main/paradox/general/stream/stream-design.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+ * supreme compositionality: combined pieces retain the function of each part
+ * exhaustive model of the domain of distributed bounded stream processing
+
+This means that we provide all the tools necessary to express any stream processing topology, that we model all the essential aspects of this domain (back-pressure, buffering, transformations, failure recovery, etc.) and that whatever the user builds is reusable in a larger context.
+
+### Akka Streams does not send dropped stream elements to the dead letter office
+
+One important consequence of offering only features that can be relied upon is the restriction that Akka Streams cannot ensure that all objects sent through a processing topology will be processed. Elements can be dropped for a number of reasons:
+
+>
```
Comment (raboof) [inline akka-docs-new/src/main/paradox/general/stream/stream-design.md]: it seems this causes the bulleted list to be indented as well as bulleted. Not sure if that makes sense to me, but it's consistent between the RST and Paradox versions

**Comment by raboof** [comment]
Work continues in `wip-akka-docs-new` instead

**Pull request closed without merging**